cfgov/jobmanager/models.py

```diff
@@ -102,8 +103,8 @@ def get_absolute_url(self):
 
     def save(self):
         if self.date_created == None:
-            self.date_created = datetime.now()
-        self.date_modified = datetime.now()
+            self.date_created = timezone.now()
+        self.date_modified = timezone.now()
```

Comment:
This doesn't significantly change the behavior of this field for legacy data (or going forward until this is moved to Wagtail) but eliminates generation of `RuntimeWarning` exceptions that complain about use of naive `DateTime`.